showcase/src/assets/examples/breadcrumb/breadcrumb.html

```diff
@@ -0,0 +1 @@
+<stark-breadcrumb breadcrumbId="breadcrumb-demonstration"></stark-breadcrumb>
```

Comment:
`breadcrumbId` is not an input in the implementation of breadcrumb component.
Could you please provide 2 examples for this component? One with passed `breadcrumbConfig` and the other without any inputs.